src/main/java/seedu/staffsnap/commons/util/StringUtil.java

```diff
@@ -31,13 +31,32 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
         checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
 
-        String preppedSentence = sentence;
-        String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
+        String[] wordsInPreppedSentence = sentence.split("\\s+");
 
         return Arrays.stream(wordsInPreppedSentence)
                 .anyMatch(sentenceWord -> sentenceWord.toLowerCase().contains(preppedWord.toLowerCase()));
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case, but a full word match is required.
+     *   <br>examples:<pre>
```

Comment:
Perhaps it is better to remove the html tags from the javadoc? 😄

Reply:
i think that the html tags are there to allow the javadoc to be rendered specifically like in this style below. Normally I won't do this, but am following the convention used in this file from #containsWordIgnoreCase

![image](https://github.com/AY2324S1-CS2103T-W08-1/tp/assets/84584280/a01316a1-7009-4171-9968-922fe8704613)

Reply:
Ohhh ok nicee 👍